### One Tailed (Left Tailed) - One-Sample Z-Test

**1 - What is it?**

A. `Left One-Tailed One-Sample Z-Test` is used to test whether a sample mean is `significantly lower` than population mean, when the `population standard deviation` is known and the `sample size is large (typically>=30)`.

**In other words, it checks:**

Is my sample average *truly smaller* than the overall population average, or is that difference just due to random variation?

**Example**

Is my small group's of average scores *really lower (significantly)* from the entire country's average score, or it just random chance.

**2 - Real-Life Example: Left Tailed One-Sample Z-Test**

`Scenario:` A teacher believes here new teaching method might have produced `lower scores` this year due to a change in syllabus.

So we wanted to test if its really decreased the scores, not just difference - So it's one-tailed test (specifically, left-tailed)

**3 - Hypotheses**

- H0 (Null Hypothesis) : No decrease in population mean
- H1 (Alternate Hypothesis) : Sample mean < Population mean (scores decreased)

This is `left-tailed` test because we're checking `less than`

**4 - Formula & Parameters**

Z = (Sample Mean - Population Mean) / (Population Std)/Sqrt(Sample Size)

Z = (x bar - μ) / (σ/sqrt(n))

**Parameters:**

- Population mean (μ) : 70
- Population std (σ) : 10
- Sample mean (x bar) : 67
- Sample size (n) : 40
- Significance level (a) : 0.05
- Confidence Interval (CI) : 0.95

Here, `populatin std is known and sample size >=30`, So we can do z-test, specifically *left one-tailed one-sample z-test*.

**5 - Python Implementation (Step-by-Step)**

In [3]:
import numpy as np
from scipy import stats

# ----------------------------------------------------------
# Step 1: Inputs - These are the numbers we already know
# ----------------------------------------------------------
population_mean = 70
population_std  = 10
sample_mean     = 67
sample_size     = 40
alpha           = 0.05

# ----------------------------------------------------------
# Step 2: Standard Error (SE)
# This tell us how the sample mean is expected to wiggle
# just because of random variation.
# ----------------------------------------------------------
standard_error = population_std / np.sqrt(sample_size)
print(f"Standard Error: {standard_error:.4f}")

# ----------------------------------------------------------
# Step 3: Z-Statistic
# This tell us how far our new mean(67) is from the old mean(70)
# Measured in units of SE.
# Bigger z = stronger evidence that the increase is real.
# ----------------------------------------------------------
z_stat = (sample_mean - population_mean)/standard_error
print(f"Z Statistics: {z_stat:.4f}")

# ----------------------------------------------------------
# Step 4: P-Value (Left-tailed)
# This answers: "if there was NO improvement,
# what is the chance of seeing a result this high purely by luck?"
# Small p-value (<alpha) = unlikely to be luck = improvement is real.
# ----------------------------------------------------------
p_value = stats.norm.cdf(z_stat)
print(f"P-Value: {p_value:.4f}")

# ----------------------------------------------------------
# Step 5: Critical Value
# This is the cutoff point for the significance.
# For a=0.05, left-tailed -> critical z=1.645
# If z_stat is above, it's considered statistically high enough.
# ----------------------------------------------------------
critical_value = stats.norm.ppf(alpha)
print(f"Critical Value: {critical_value:.4f}")

# ----------------------------------------------------------
# Step 6: Decision Based on P-Value
# ----------------------------------------------------------
print("\n--------- Decision Based on P-Value ------------")
if p_value < alpha:
    print("Reject H0: New syllabus made an negative impact in students score.")
else:
    print("Fail to Reject H0: There were no strong evidence to prove the difference is real.")

# ----------------------------------------------------------
# Step 7: Decision Based on Z-Value
# ----------------------------------------------------------
print("\n---------- Decision Based on P-Value -----------")
if z_stat < critical_value:
    print("Reject H0: New syllabus made an negative impact in students score.")
else:
    print("Fail to Reject H0: There were no strong evidence to prove the difference is real.")

Standard Error: 1.5811
Z Statistics: -1.8974
P-Value: 0.0289
Critical Value: -1.6449

--------- Decision Based on P-Value ------------
Reject H0: New syllabus made an negative impact in students score.

---------- Decision Based on P-Value -----------
Reject H0: New syllabus made an negative impact in students score.


**6 - Business Interpretation**

- **Z = 1.90** -> The sample mean is 1.9 standard error below the population mean.
- **p = 0.029** -> There's only about a 2.9% probability of observing such a low average if the true mean were still 70.

**Conclusion:**
This suggests that decrease in score is **statistically significant**, meaning it's unlikely due to random variation.
The new syllabus **likely had a negative impact** on students performance.